README.md

```diff
@@ -45,8 +91,9 @@ Execute installation script
     * Set the hostname of the endpoints correctly as per your docker setup
     * Now generate a DID, create a credential schema and create an issuance registry
         * take note of `$.schema[0].author`  and  `$.schema[0].id` from the create credential schema request
-6. Create a folder with name loader_path [here](docker-compose/docker-compose-certify).
-7. Add the jar file of Digital Credential Stack(DCS) plugin implementations for eSignet and certify:
+6. Change the variable `active_profile_env` in [esignet](docker-compose/docker-compose-certify/docker-compose.yml#L79) and [certify](docker-compose/docker-compose-certify/docker-compose.yml#L103) to `active_profile_env=default,sunbird-insurance`
+7. Create a folder with name loader_path [here](docker-compose/docker-compose-certify).
```

Comment:
In the mock steps, this and few steps are optional since it takes the jar file artifactory. In case of sunbird also can we not keep these steps optional ?

Reply:
kept these as optional steps in sunbird